packages/tldraw/src/lib/ui/components/MainMenu/DefaultMainMenuContent.tsx

```diff
@@ -158,6 +160,32 @@ export function ExtrasGroup() {
 	)
 }
 
+/** @public @react */
+export function HelpGroup() {
+	function openUrl(url: string) {
+		window.open(url, '_blank')
+	}
+
+	return (
+		<TldrawUiMenuGroup id="help">
+			<TldrawUiMenuSubmenu id="help" label="menu.help">
+				<TldrawUiMenuGroup id="help-actions">
+					<KeyboardShortcutsMenuItem />
+					<TldrawUiMenuItem
+						id="about"
+						label="help-menu.docs"
```

Comment:
Should we rename these labels? If we started from scratch we'd probably use `menu.docs`. Feels like it would make it easier for people to understand what the label applies to if we have consistent naming.

Reply:
i did start with `menu.docs` but then i realized that it should live alongside its peers:
```
        "help-menu.about": "About",
	"help-menu.docs": "Documentation & API",
	"help-menu.discord": "Discord",
	"help-menu.github": "GitHub",
	"help-menu.keyboard-shortcuts": "Keyboard shortcuts",
	"help-menu.twitter": "Twitter",
```

<img width="307" alt="Screenshot 2024-07-02 at 17 45 38" src="https://github.com/tldraw/tldraw/assets/469604/c44c89ca-7a3a-45fa-9514-afc4d09653dc">

but i'm not too opinionated on it. i'll land this PR and i don't mind if you change it in the base PR :)